### PR TITLE
RT490: Add supported charset

### PR DESCRIPTION
**Pull request by pbarrette**
Added additional characters supported by the RT-490 display. Tested against MMLradio JC-8629.

Fixes issues 11020 and 11309.

### CODE REVIEW

File: chirp/drivers/radtel_rt490.py [hunk 1]
```diff
@@ -421,6 +421,9 @@ class RT490Radio(chirp_common.CloneModeRadio):
                (0x3FC0, 0x4000)
               ]
 
+    _valid_chars = chirp_common.CHARSET_ALPHANUMERIC + \
+        "`~!@#$%^&*()-=_+[]\\{}|;':\",./<>?"
+
```
Comment (kk7ds) [inline chirp/drivers/radtel_rt490.py:426]: This isn't used anywhere so this won't have any effect. I assume you want to set `valid_characters` in `get_features()` ?

Comment (pbarrette) [reply chirp/drivers/radtel_rt490.py:426]: And yet it works, so I don't know what to tell you.

I took the example directly from mml_jc8810.py

Comment (kk7ds) [reply chirp/drivers/radtel_rt490.py:426]: It really doesn't. I changed it to just `'a'` and I can still put in names like "FOO". With and without your patch, I can only put in uppercase characters.

The `mmp_jc8810.py` actually references it exactly where I said, which is why it works there and does not work here:

https://github.com/kk7ds/chirp/blob/master/chirp/drivers/mml_jc8810.py#L470

Comment (pbarrette) [reply chirp/drivers/radtel_rt490.py:426]: Well, crap. I clearly missed that while I was trying to not re-contaminate with the NEEDS_COMPAT_SERIAL that still existed in my original file.

**Review by kk7ds — COMMENTED** [review]
Can you squash these into a single commit (per the PR guidelines template)? The delta goes to the mailing list on every build and there's no need to tell 2000 people about both changes, just the fix :)

Thanks!

**Comment by kk7ds** [comment]
And also please give that commit a useful commit message so the recipients know what you fixed. Please also (per the guidelines) put `Fixes #11020` and `Fixes #11309` in the commit message so it links the bugs as well.

**Comment by pbarrette** [comment]
Done.

**Pull request closed without merging**